quicklogic/common/toolchain_wrappers/symbiflow_analysis

```diff
@@ -11,6 +11,6 @@ parse_args $@
 
 export OUT_NOISY_WARNINGS=noisy_warnings-${DEVICE}_analysis.log
 
-run_vpr --analysis --gen_post_synthesis_netlist on
+run_vpr --analysis --gen_post_synthesis_netlist on --verify_file_digests off
```

Comment:
I presume that we have to disable that because the packed netlist contains SHA of the other arch.xml and they do not match?

Reply:
Indeed, another solution might be to adjust the SHA, but it would be yet another step in the toolchain that we can avoid with this option.

Reply:
Ok.